runsc/cgroup/BUILD

```diff
@@ -4,12 +4,19 @@ package(licenses = ["notice"])
 
 go_library(
     name = "cgroup",
-    srcs = ["cgroup.go"],
+    srcs = [
+        "cgroup.go",
+        "cgroup_v2.go"
+    ],
     visibility = ["//:sandbox"],
     deps = [
         "//pkg/cleanup",
         "//pkg/log",
         "@com_github_cenkalti_backoff//:go_default_library",
+        "@com_github_opencontainers_runc//libcontainer/cgroups:go_default_library",
+        "@com_github_opencontainers_runc//libcontainer/cgroups/fs2:go_default_library",
+        "@com_github_opencontainers_runc//libcontainer/configs:go_default_library",
+        "@com_github_opencontainers_runc//libcontainer/utils:go_default_library",
```

Comment:
Although it's highly desirable to reuse existing code, we try to [limit the number of external packages](https://gvisor.dev/docs/architecture_guide/security/#principles-defense-in-depth) used in runsc. We control very tightly dependencies in runsc for multiple reasons:

1. All code that is used to build runsc must be audited and carefully reviewed
2. External dependencies don't work very well with Google's monorepo and are really hard to maintain
3. runsc cannot use cgo

Thus it's preferable to not use libcontainer in here and implement the code that interfaces with cgroups V2 directly here.